core/src/test/scala/akka/persistence/cassandra/CassandraSpec.scala

```diff
@@ -38,28 +39,57 @@ object CassandraSpec {
     reduced.head.replaceFirst(""".*\.""", "").replaceAll("[^a-zA-Z_0-9]", "_")
   }
 
-  def configWithKeyspace(name: String) = ConfigFactory.parseString(
+  def configWithKeyspace(journalKeyspace: String, snapshotStoreKeyspace: String, port: Int): Config = ConfigFactory.parseString(
     s"""
-      cassandra-journal.keyspace = $name
-       cassandra-query-journal = {
-         first-time-bucket = "${today.minusHours(2).format(query.firstBucketFormat)}"
```

Comment:
isn't it good to define this?

Reply:
Yes, i had moved it to CassandraLifecycle but got lost at some point.

It needs to be in a fallbackConfig rather than this section otherwise it overrides tests that rely on the values they set. Will do that now